src/plugins/multimedia/reports/a11y-1-youtube.json

```
@@ -0,0 +1,1962 @@
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "earl:mode": "earl:automatic",
+      "@type": [
+        "earl:TestResult",
+        "earl:Assertion"
+      ]
+    },
+    {
+      "earl:test": "WCAG21:resize-text",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "earl:mode": "earl:automatic",
+      "@type": [
+        "earl:TestResult",
+        "earl:Assertion"
+      ]
+    },
+    {
+      "earl:test": "WCAG21:image-of-text",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "earl:mode": "earl:automatic",
+      "@type": [
+        "earl:TestResult",
+        "earl:Assertion"
+      ]
+    },
+    {
+      "earl:test": "WCAG21:reflow",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "earl:mode": "earl:automatic",
+      "@type": [
+        "earl:TestResult",
+        "earl:Assertion"
+      ]
+    },
+    {
+      "earl:test": "WCAG21:non-text-contrast",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "earl:mode": "earl:automatic",
+      "@type": [
+        "earl:TestResult",
+        "earl:Assertion"
+      ]
+    },
+    {
+      "earl:test": "WCAG21:text-spacing",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "earl:mode": "earl:automatic",
+      "@type": [
+        "earl:TestResult",
+        "earl:Assertion"
+      ]
+    },
+    {
+      "earl:test": "WCAG21:content-on-hover-or-focus",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "earl:mode": "earl:automatic",
+      "@type": [
+        "earl:TestResult",
+        "earl:Assertion"
+      ]
+    },
+    {
+      "earl:test": "WCAG21:keyboard",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:fail",
+      "dct:description": "The Youtube multimedia plugin contains elements such as watch later and share which are not focusable through keyboard navigation only. Therefore the plugin fails the assessment.",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "earl:mode": "earl:automatic",
```

The outcome technical name for the fail is: `earl:failed` as documented here: https://www.w3.org/TR/EARL/#OutcomeValue

```suggestion
      "earl:outcome": "earl:failed",
      "dct:description": "The Youtube multimedia plugin contains elements such as watch later and share which are not focusable through keyboard navigation only. Therefore the plugin fails the assessment.",
      "acr:severity": "acr:noSeverity",
      "acr:relevancy": "wbv:acr#noRelevancy",
      "earl:mode": "earl:semiAuto",
```